### DISCUSSION TITLE
Should I use Ragel 7?

### DISCUSSION THREAD

**Discussion by burdiyan** [adrian-thurston/ragel#96] [Q&A]
Despite being in development (at a first glance might even seem dead), I wonder if people tend to use Ragel 6 or 7 these days? Does Ragel 7 have some major known bugs? I'm interested in 7 mainly because it adds more target languages which I wanted.

**Comment by redmeadowman** [comment]
I have been using Ragel 7 in Fedora 36 and found the the “include” statement crashes Ragel every time. I have to combine source files external to Ragel. Plan to try include on Ragel 6 and see if it suffers from the same issue.

**Comment by rofl0r** [comment]
ragel 6.10 works (almost) rock-solid, but 7 seems far from stable juding from bug reports here. unfortunately most distros tend to always ship latest versions of anything, regardless of whether it works better or worse.
